### PR TITLE
add "ChainerMN on Kubernetes with GPUs"

### PR DESCRIPTION
**Pull request by everpeace**
Hi Chainer developers!

I would like to post an article how to run ChainerMN on Kubernetes with GPUs.  Kubernetes is today the most popular container orchestrator and I feel it would be de-facto infrastructure for machine learning on cloud environment.  

But running ChainerMN workload on kubernetes is not straight forward because it requires us to setup OpenMPI cluster on kubernetes cluster.

So, I believe that this topic would be useful for users who wanted to run ChainerMN on kubernetes.

__NOTE:  I would like to publish this article on 5/10 JST for internal reason.  Thanks in advance.__

### CODE REVIEW

File: _posts/2018-05-10-chainermn-on-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  ompi_info --parsable --all | grep -q "mpi_built_with_cuda_support:value:true"
+
+# Install NCCL2
+RUN wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb && \
+    dpkg -i nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb && \
+    rm nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb && \
+    apt-get update && \
+    apt-get install -yq --no-install-recommends \
+      libnccl-dev=${NCCL_PACKAGE_VERSION} libnccl2=${NCCL_PACKAGE_VERSION} && \
+  rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
```
Comment (mitmul) [inline _posts/2018-05-10-chainermn-on-kubernetes.md]: Do you really need to install NCCL2 itself? When you install cupy with `pip install cupy-cuda80` command, it also brings built binaries of cuDNN, NCCL, and CUDA.

File: _posts/2018-05-10-chainermn-on-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+When generating a component, we can specify several _parameters_.  In this example, we specify
+
+- `train-mnist` for its name,
+- `4` workers,
+- `1` GPU for each worker, and
+- launching `mpiexec ... train_mnist.py` scirpt for `exec` param
+
+And then, `ks apply` command deploy our [OpenMPI](https://www.open-mpi.org/)  cluster on [Kubernetes](https://kubernetes.io/) cluster.
+
+_Please be advised that this step requires an authorization to create service accounts and cluster role bindings for "view" cluster role.  If you didn't have such authorization, you will have to ask your administrator create a service account which is granted 'get' verb for 'pods' resources. If such service account was ready, you then will set it to `serviceAccountName` param of `train-mnist` component._
```
Comment (mitmul) [inline _posts/2018-05-10-chainermn-on-kubernetes.md]: ask your administrator create -> ask your administrator to create

**Comment by everpeace** [comment]
@mitmul Thanks for your review!! I pushed fixes.  Could you review again 🙇 ?

**Comment by mitmul** [comment]
LGTM

**Pull request merged**